submissions/use_external_data/estimator.py

```diff
@@ -13,11 +14,12 @@ def _merge_external_data(X):
         os.path.dirname(__file__), 'external_data.csv'
     )
     # Make sure that DateOfDeparture is of dtype datetime
+    X = X.copy()  # modify a copy of X
```

Comment:
Why a copy?

Reply:
I get pandas warnnings otherwise as I modify a slice of a dataframe